### PR TITLE
fix(reload): working on dashboard reload bug

### PR DESCRIPTION
**Pull request by Shaharyar-saleem**
All the issues have solved

### CODE REVIEW

**Comment by bobachef** [comment]
> working is in process getting following ERROR:
> SyntaxError: Unexpected token o in JSON at position 1

is this still error that you're getting?

**Comment by Shaharyar-saleem** [comment]
> > working is in process getting following ERROR:
> > SyntaxError: Unexpected token o in JSON at position 1
> 
> is this still error that you're getting?

No, i am not getting this error now.

**Comment by Shaharyar-saleem** [comment]
@bobachef I need your help in the logout part

File: assets/js/connectWallet.js [hunk 1]
```diff
@@ -721,7 +730,12 @@ function validateErcAddress(address) {
   return false;
 }
 
+
 // Logout
-function logOut() {
-  isConnected = false;
+async function logOut() {
+    await web3Modal.clearCachedProvider();
+    sessionStorage.clear();
```
[the remark this answers is not in the document]
Comment (Shaharyar-saleem) [reply assets/js/connectWallet.js]: done

**Review by bobachef — CHANGES_REQUESTED** [review]
Still way too many issues. Your testing should have found this. Please please please test your code before submitting this.

1. Disclaimer modal needs to open in center of view
2. Disclaimer needs both boxes checked and submitted before being able to proceed.
3. The connect button doesn't work properly and brings you to metamask site instead of the modal.
4. Please go through and find more bugs on your own.

**Comment by Shaharyar-saleem** [comment]
> Still way too many issues. Your testing should have found this. Please please please test your code before submitting this.
> 
> 1. Disclaimer modal needs to open in center of view
> 2. Disclaimer needs both boxes checked and submitted before being able to proceed.
> 3. The connect button doesn't work properly and brings you to metamask site instead of the modal. This is probably because you have the modal opening automatically.
> 4. Please go through and find more bugs on your own.

All points followed

**Review by bobachef — APPROVED** [review]
lgtm

**Pull request merged**